### PR TITLE
[DNM] DeletefilesInRanges + trigger next

### PR DESCRIPTION
**Pull request by Connor1996**
only for test

### CODE REVIEW

File: src/blob_storage.cc [hunk 1]
```diff
@@ -121,8 +121,9 @@ void BlobStorage::ComputeGCScore() {
     gc_score_.push_back({});
     auto& gcs = gc_score_.back();
     gcs.file_number = file.first;
-    if (file.second->file_size() < cf_options_.merge_small_file_threshold) {
-      gcs.score = 1;
+    if (file.second->file_size() <= cf_options_.merge_small_file_threshold ||
+        file.second->gc_mark()) {
+      gcs.score = cf_options_.blob_file_discardable_ratio;
```
Comment (yiwu-arbug) [inline src/blob_storage.cc]: why not set to 1 as before?

File: src/blob_format.h [hunk 2]
```diff
@@ -151,7 +155,7 @@ class BlobFileMeta {
   FileState state_{FileState::kInit};
 
   uint64_t discardable_size_{0};
-  //  bool marked_for_gc_{false};
+  bool gc_mark_{false};
```
Comment (yiwu-arbug) [inline src/blob_format.h]: Add a comment describe meaning of `gc_mark_`

File: src/blob_gc_picker.cc [hunk 3]
```diff
@@ -57,8 +57,10 @@ std::unique_ptr<BlobGC> BasicBlobGCPicker::PickBlobGC(
         stop_picking = true;
       }
     } else {
-      if (blob_file->GetDiscardableRatio() >=
-          cf_options_.blob_file_discardable_ratio) {
+      if (blob_file->file_size() <= cf_options_.merge_small_file_threshold ||
```
Comment (yiwu-arbug) [inline src/blob_gc_picker.cc]: Does it mean we are more aggressively merging small files than before?

File: src/blob_storage.cc (hunk 1 above)
Comment (Connor1996) [reply src/blob_storage.cc]: To make them low priority. We want to merge blob files that exceed the discardable ratio first.

File: src/blob_gc_picker.cc (hunk 3 above)
Comment (Connor1996) [reply src/blob_gc_picker.cc]: Yep, merging small file doesn't work as expected before cause `DoSample` doesn't consider `merge_small_file_threshold`.

Comment (yiwu-arbug) [reply src/blob_gc_picker.cc]: I see.

File: src/blob_storage.cc (hunk 1 above)
Comment (yiwu-arbug) [reply src/blob_storage.cc]: make sense. better to add an inline comment.

**Pull request closed without merging**